components/tools/OmeroPy/src/omero/plugins/chgrp.py

```diff
@@ -18,23 +18,30 @@
 
 HELP = """Move data between groups
 
-Example Usage:
+Move entire graphs of data based on the ID of the top-node.
```

Comment:
Hmm, I think the second line if more of an emphasis of what happens though maybe it could be imporoved?